test/test_pkg_test.py

```diff
@@ -86,3 +89,31 @@ def test_pkg_test_custom_base_image():
     """
     build_package = _build_pkg(RECIPE_CUSTOM_BASE)
     res = pkg_test.test_package(build_package, base_image='debian:latest')
+
+
+@pytest.mark.skipif(SKIP_OSX, reason="skipping on osx")
+def test_pkg_test_conda_image():
+    """
+    Running a mulled-build test with a non-default conda image.
+    """
+    # Inspects the installing conda version by writing $PREFIX/conda-version as
+    # a post-link step -- but only if we are actually doing mulled tests, i.e.,
+    # when $PREFIX == /usr/local.
+    recipe = dedent("""
+        one:
+          meta.yaml: |
+            package:
+              name: one
+              version: 0.1
+            test:
+              commands:
+                - '[ "${PREFIX%%/}" != "/usr/local" ] || cat "/usr/local/conda-version"'
+                - '[ "${PREFIX%%/}" != "/usr/local" ] || grep -F 4.3.11 "/usr/local/conda-version"'
+          post-link.sh: |
```

Comment:
Very clever! :+1:

Reply:
The `${PREFIX%%/}` didn't work because of the `${PREFIX}` string replacement in `pkg_test.get_tests`. Fixed with a new commit.